package.json

```
@@ -1,8 +1,12 @@
 {
-  "name": "EngageSphere-Test-Design-Masterclass-TAT",
+  "name": "engagesphere-test-design-masterclass-tat",
```

Algum motivo para mudar o nome do projeto?

```suggestion
  "name": "EngageSphere-Test-Design-Masterclass-TAT",
```